pom.xml

```diff
@@ -23,48 +23,102 @@
             <groupId>org.esa.snap</groupId>
             <artifactId>ceres-core</artifactId>
 			<version>${snap.version}</version>
+			<exclusions>
```

Comment:
Why are you excluding the tools artifact? This should actually not cause any problem when using OpenJDK 8.

Reply:
This is code I was given (v1.2.0), and is being used by multiple users.

I'll be creating a branch for v1.2.1 and I will remove the tools artifact exclusion then.

Many thanks